lib/param_parsing/toml_test.go

```diff
@@ -0,0 +1,33 @@
+package param_parsing
+
+import (
+	"testing"
+)
+
+func prepare() Params {
+	var params Params
+	params.ChDirPath = "../../test-data/test_tfswitchtoml"
+	return params
+}
+
+func TestGetParamsTOML_BinaryPath(t *testing.T) {
+	expected := "/usr/local/bin/terraform_from_toml"
+	params := prepare()
+	params = GetParamsTOML(params)
+	if params.CustomBinaryPath != expected {
+		t.Log("Actual:", params.CustomBinaryPath)
+		t.Log("Expected:", expected)
+		t.Error("BinaryPath not matching")
```

Comment:
I think it's worth to keep this consistent with other similar functions by using single `t.Error()` line